EXTRA_DEPENDENCIES.rst

```diff
@@ -0,0 +1,19 @@
+=========================
```

Comment:
Rather than combining a description of extra dependencies + installation instructions in a single INSTALLATION.rst file linked-to by the quickstart, I kept the existing installation instructions inline in the quickstart for visibility (and because they have some nice properties like code tabs for installing Python vs R)

Therefore I ended up calling this file EXTRA_DEPENDENCIES.rst & focusing only on describing steps for installing extra dependencies